src/tenders/models.py

```diff
@@ -0,0 +1,25 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Tender(models.Model):
+    class StatusChoices(models.IntegerChoices):
+        NEW = 0, "New"
+        EXECUTED = 1, "Executed"
+
+    buyer = models.ForeignKey(to="accounts.BuyerProfile", related_name="tender", on_delete=models.CASCADE)
+    end_day = models.DateField(null=True)
```

Comment:
null=True and blank=True: If a field can be empty or null, it's better to use both null=True and blank=True instead of only one of them. null=True allows the field to be null in the database, and blank=True allows the form to be submitted without the field being required.

Reply:
Or here you can use as a default value current date + timedelta.

Reply:
For example + 14 days